lambdas/service/app.py

```diff
@@ -213,12 +213,33 @@ def validate_repository_search(params, **validators):
         'search_after_uid': str,
         'search_before': str,
         'search_before_uid': str,
-        'size': int,
+        'size': validate_size,
         'sort': validate_facet,
         **validators
     })
 
 
+def validate_size(size):
+    """
+    >>> validate_size('999')
+
+    >>> validate_size('100000')
+    Traceback (most recent call last):
+    ...
+    chalice.app.BadRequestError: BadRequestError: Invalid size value, must be less than 1000
```

Comment:
I'd add coverage for the remaining cases, too.

https://coveralls.io/builds/26049933/source?filename=lambdas%2Fservice%2Fapp.py#L219
https://coveralls.io/builds/26049933/source?filename=lambdas%2Fservice%2Fapp.py#L221